.github/workflows/deployDemo.yml

```diff
@@ -47,7 +47,9 @@ jobs:
       - run: |
           echo "DOCKER_REGISTRY=${{ steps.login-ecr.outputs.registry }}" >> $GITHUB_ENV
           echo "REPO=demo" >> $GITHUB_ENV
-          echo "TAG=${BRANCH}-b${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
+          TAG=$(TAG=${BRANCH}-b${GITHUB_RUN_NUMBER} task docker:clean-tag)
```

Comment:
i really dont love this. the only reason to have set the "corrected" tag back to the gha env is to reference the artifact in the slack notification that follows. if we did the slack notification ourselves in Task we could drop this entirely.